updatedata.go

```diff
@@ -0,0 +1,74 @@
+package main
+
+import (
+	"context"
+	"encoding/json"
+	"log"
+	"net/http"
+
+	"go.mongodb.org/mongo-driver/bson"
+	"go.mongodb.org/mongo-driver/mongo"
+	"go.mongodb.org/mongo-driver/mongo/options"
+)
+
+type address struct {
+	Street  string `bson:"Street"`
+	City    string `bson:"City"`
+	State   string `bson:"State"`
+	Country string `bson:"Country"`
+}
+
+type User struct {
```

Comment:
we should always keep the variables in main package lowercase, uppercase first letter means we can use it from outside the package, but it is not useful in case of main package